curp/src/server/curp_node.rs

```diff
@@ -847,6 +867,19 @@ impl<C: 'static + Command, RC: RoleChange + 'static> CurpNode<C, RC> {
     pub(super) fn shutdown_listener(&self) -> shutdown::Listener {
         self.curp.shutdown_listener()
     }
+
+    /// Check cluster version and return new cluster
+    fn check_cluster_version(&self, client_cluster_version: u64) -> Result<(), CurpError> {
+        let server_cluster_version = self.curp.cluster().cluster_version();
+        if client_cluster_version != server_cluster_version {
+            debug!(
```

Comment:
Should it be a warning level?

Reply:
FYI: https://github.com/xline-kv/Xline/pull/460#discussion_r1361423702